### ISSUE TITLE
Add Support for Long in equalTo

### ISSUE THREAD

**Issue by bennidi** [marcinziolo/kotlin-wiremock#11]
The class `ContainsSteps` does not offer a method overload that accepts Long. This can easily be added. If you want I can make a PR.

**Comment by marcinziolo** [comment]
Yep, go ahead :) thanks. I will release a new version when PR is merged.

**Comment by marcinziolo** [comment]
Thanks for suggestion, added to new version 1.0.3

**Comment by bennidi** [comment]
Nice. Thank you, I was admittedly too slow. Thumbs up for you!
